Client/Frontend/Widgets/Menu/MenuViewController.swift

```diff
@@ -160,7 +160,17 @@ class MenuViewController: UIViewController {
 
     override func viewWillLayoutSubviews() {
         super.viewWillLayoutSubviews()
+        relayoutMenuAndSetSize()
+    }
+
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+        relayoutMenuAndSetSize()
```

Comment:
OK, I've managed to test now. Yes, this causes the layout code for `MenuView` to get called up to 4 times every time the menu is displayed. 

I achieved the same effect (correct menu display) on iOS10 & iOS9 by moving this :

```

        if presentationStyle == .Popover {
            self.preferredContentSize = CGSizeMake(view.bounds.size.width, menuView.bounds.size.height)
        }
```

to `viewDidAppear` rather than `viewWillLayoutSubviews`, which I think may be a less invasive fix.